### ISSUE TITLE
Windows 10 64bit

### ISSUE THREAD

**Issue by Firefoxin** [pipe01/legendary-rune-maker#8]
Hi , i can't run program in windows 10 (64bit) build 17314

download link: https://ci.appveyor.com/project/pipe01/legendary-rune-maker/build/artifacts **(1.0.71)**

also it doesn't show me any error message.

Thanks

**Comment by pipe01** [comment]
Do you have .NET Framework 4.6 installed?

**Comment by Firefoxin** [comment]
.NET Framework 4.6.2 or a later update is already installed on computer.

**Comment by pipe01** [comment]
Do you see any windows at all?

**Comment by Firefoxin** [comment]
Yes, first time I open application in administrator mode windows show me blue screen "**Do You Trust unsigned application?**"
some message like this picture [[LINK](http://eeme.co/static/q_coding/windows_install/launch_02.jpg)] , after that i click on **Run Anyway** and then nothing happens.

**Comment by pipe01** [comment]
Did you download and extract the zip file?

**Comment by Firefoxin** [comment]
if I download and run .exe File : [[Link]](http://oi65.tinypic.com/oaok11.jpg) > Run anyway
and
If I download and run .Zip file : [[Link]](http://oi64.tinypic.com/110lsa0.jpg)

**Comment by pipe01** [comment]
You should always download the zip file. If you run the exe in the zip
file, does it show on task manager?

**Comment by Firefoxin** [comment]
OH ! don't now how but its **OK** now !!!! I'm not sure but it toke more than 30min after I run the zip file.
Thanks for your support 👍
